<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelFox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>741</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>The quick brown fox likes CSE 335</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>585</width>
      <height>126</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="sizetype">
         <property name="text">
          <string>Font Size Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sizetype_2">
         <property name="text">
          <string>Font Size Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="family">
         <property name="text">
          <string>Family</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fontsize">
         <property name="text">
          <string>Font Size</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="LineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="combobox" name="comboBox">
         <item>
          <property name="text">
           <string>small</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>medium</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>big</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="checkbox" name="AcheckBox">
           <property name="text">
            <string>Arial</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="checkbox" name="TcheckBox">
           <property name="text">
            <string>Times New Roman</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="checkbox" name="CcheckBox">
           <property name="text">
            <string>Comic Sans MS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="slider" name="horizontalSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>combobox</class>
   <extends>QComboBox</extends>
   <header location="global">combobox.h</header>
  </customwidget>
  <customwidget>
   <class>checkbox</class>
   <extends>QRadioButton</extends>
   <header location="global">checkbox.h</header>
  </customwidget>
  <customwidget>
   <class>slider</class>
   <extends>QSlider</extends>
   <header location="global">slider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
